contracts/Heiswap.sol

```diff
@@ -285,7 +285,7 @@ contract Heiswap {
 
     // Gets all addresses in a Ring
     // Converting to Bytes32 cause web3.js has a bug that doesn't convert
-    // Big Num correctly....
+    // BigNum correctly....
```

Comment:
lol

Reply:
It's true

Reply:
@kendricktan is there an issue on Web3.js for it?

Reply:
@adrianmcli it only occurs on some big integers (I.e < JS native 32 bit integers)